rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -244,6 +250,16 @@ def decode_by_address_rules(
 
         return result
 
+    def run_all_post_decoding_rules(
+            self,
+            transaction: EvmTransaction,
+            decoded_events: list[HistoryBaseEntry],
+            action_items: list[ActionItem],
```

Comment:
action items are not needed here, since they are supposed to be only used during the decoding step.